pylana/api.py

```diff
@@ -11,7 +11,12 @@
 
 
 def _create_authorization_header(token: str) -> dict:
-    return {"Authorization": f"API-Key {token}"}
+    token_bearer = "Bearer"
+    token_api = "API"
+    if token[:(len(token_api))] == token_api or token[:(len(token_bearer))] == token_bearer:
+        return {"Authorization": f"{token}"}
+    else:
+        return {"Authorization": f""}
```

Comment:
The f-string shouldn’t be used here as there are no placeholders.

Reply:
The idea would be that the header still gets created with an _empty_ish content and the authentication will certainly fail on that. So the mining back-end cannot get accessed in case an unsupported content comes in from a user.

Also, I would like to avoid hanging _else_ branches in an if expression, so the function is surely total and does an actual check.

Reply:
I've removed the `f`. Thanks!